lib/src/animated_sampler.dart

```diff
@@ -157,58 +154,54 @@ class _RenderShaderSamplerBuilderWidget extends RenderProxyBox {
     markNeedsPaint();
   }
 
-  ui.Image? _childRaster;
-
-  @override
-  void detach() {
-    _childRaster?.dispose();
-    _childRaster = null;
-    super.detach();
-  }
-
   @override
-  void dispose() {
-    _childRaster?.dispose();
-    _childRaster = null;
-    super.dispose();
-  }
+  bool get alwaysNeedsCompositing => true;
```

Comment:
I would change this to

`bool get alwaysNeedsCompositing => enabled`

And then update the enabled setter to call `markNeedsCompositingBitsUpdate` if the value changes

Reply:
Done